be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -706,6 +730,19 @@ Status HashJoinNode::init(const TPlanNode& tnode, RuntimeState* state) {
                                                                    filter_desc, state->query_options()));
     }
 
+    // init left/right output slots flags
```

Comment:
Could you please add a example to illustrate the value of the '_left_output_slot_flags' and '_left_output_slot_flags'